app/src/main/java/com/babylon/wallet/android/domain/usecases/BuildDappResponseUseCase.kt

```diff
@@ -329,7 +330,13 @@ class BuildUnauthorizedDappResponseUseCase @Inject constructor(
                     items = WalletToDappInteractionResponseItems.UnauthorizedRequest(
                         v1 = WalletToDappInteractionUnauthorizedRequestResponseItems(
                             oneTimeAccounts = oneTimeAccountsResponseItem.getOrNull(),
-                            oneTimePersonaData = oneTimePersonaData?.toWalletToDappInteractionPersonaDataRequestResponseItem()
+                            oneTimePersonaData = oneTimePersonaData?.toWalletToDappInteractionPersonaDataRequestResponseItem(),
+                            // TODO this should be replaced when merging the proof of ownership feature
```

Comment:
actually this will go away after the discussion we had on Friday's demo. But until then let's keep it as it is here.